src/EnergyPlus/EconomicTariff.cc

```diff
@@ -4083,67 +4080,97 @@ namespace EnergyPlus::EconomicTariff {
                 WriteReportHeaders(state, "Economics Results Summary Report", "Entire Facility", OutputProcessor::StoreType::Averaged);
                 elecFacilMeter = GetMeterIndex(state, "ELECTRICITY:FACILITY");
                 gasFacilMeter = GetMeterIndex(state, "NATURALGAS:FACILITY");
-                //---- Annual Summary
-                rowHead.allocate(3);
-                columnHead.allocate(4);
-                columnWidth.allocate(4);
-                tableBody.allocate(4, 3);
-                tableBody = "";
-                columnHead(1) = "Electricity";
-                columnHead(2) = "Natural Gas";
-                columnHead(3) = "Other";
-                columnHead(4) = "Total";
-                rowHead(1) = "Cost [~~$~~]";
-                rowHead(2) = "Cost per Total Building Area " + perAreaUnitName;
-                rowHead(3) = "Cost per Net Conditioned Building Area " + perAreaUnitName;
-                elecTotalCost = 0.0;
-                gasTotalCost = 0.0;
-                otherTotalCost = 0.0;
-                allTotalCost = 0.0;
-                for (iTariff = 1; iTariff <= state.dataEconTariff->numTariff; ++iTariff) {
-                    if (tariff(iTariff).isSelected) {
-                        allTotalCost += tariff(iTariff).totalAnnualCost;
-                        if (tariff(iTariff).kindElectricMtr >= kindMeterElecSimple) {
-                            elecTotalCost += tariff(iTariff).totalAnnualCost;
-                        } else if (tariff(iTariff).reportMeterIndx == gasFacilMeter) {
-                            gasTotalCost += tariff(iTariff).totalAnnualCost;
-                        } else {
-                            otherTotalCost += tariff(iTariff).totalAnnualCost;
-                            // removed because this was confusing        columnHead(3) = tariff(iTariff)%reportMeter
+
+                for (int iUnitSystem = 0; iUnitSystem <= 1; iUnitSystem++) {
+                    OutputReportTabular::iUnitsStyle unitsStyle_cur = state.dataOutRptTab->unitsStyle;
+                    bool produceTabular = true;
+                    bool produceSQLite = false;
+                    if (produceDualUnitsFlags(iUnitSystem,
+                                              state.dataOutRptTab->unitsStyle,
+                                              state.dataOutRptTab->unitsStyle_SQLite,
+                                              unitsStyle_cur,
+                                              produceTabular,
+                                              produceSQLite))
+                        break;
```

Comment:
Indentation here makes this difficult to read.  Please use braces with IFs even on single statement blocks like this.  OK for now, just use the braces in the future.

Reply:
Thanks!